templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+  </form>

```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.